dfget/config/supernode_value_test.go

```diff
@@ -98,3 +98,14 @@ func (suit *SupernodeValueSuite) TestHandleNodes() {
 		}
 	}
 }
+
+func (suit *SupernodeValueSuite) TestNodeWightSlice2StringSlice() {
```

Comment:
I think the `Wight` should be the word `Weight`, do you think so ? @Starnop @lowzj 
If that, would change all the `Wight` into `Weight`in file https://github.com/dragonflyoss/Dragonfly/blob/b32d1ca03b4111978b2d8a0f398704946e071d99/dfget/config/supernode_value.go and some other files?

Reply:
I think this should be a typo, it should be Weight, not Wight.

Reply:
Could you add a pull request to change them all?

Reply:
Done, PTAL.

Reply:
Thank you so much.